RULES.md

```diff
@@ -149,8 +149,20 @@ your code.
   })
   ```
 
-* **Always prefix browser globals** with `window.`.<br>
-  Exceptions are: `document`, `console` and `navigator`.
+* **Declare browser globals** with a `/* global */` comment.<br>
+  Exceptions are: `window`, `document` and `navigator`.<br>
+  Prevents accidental use of poorly-named browser globals like `open`, `length`,
+  `event`, and `name`.
```

Comment:
I'd remove "poorly-named" in order to sound less judgmental.

Also fun fact: any element with an `id` also creates a global variable! Great source of nasty bugs. 😛